[02/Aug/2016 02:06:55] ERROR [django.request:231] Internal Server Error: /help
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 318, in help
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[02/Aug/2016 02:06:57] ERROR [django.request:231] Internal Server Error: /help
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 318, in help
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[02/Aug/2016 02:06:58] ERROR [django.request:231] Internal Server Error: /help
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 318, in help
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[02/Aug/2016 02:07:03] ERROR [django.request:231] Internal Server Error: /help
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 318, in help
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[02/Aug/2016 02:07:06] ERROR [django.request:231] Internal Server Error: /help
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 318, in help
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'title' appears more than once
[02/Aug/2016 02:27:09] ERROR [django.request:231] Internal Server Error: /all-options
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 61, in all_options
    template = loader.get_template('all-options.html')
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 288, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 344, in invalid_block_tag
    raise self.error(token, "Invalid block tag: '%s'" % command)
TemplateSyntaxError: Invalid block tag: 'active_tab='options''
[02/Aug/2016 02:27:47] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 02:49:32] WARNING [django.request:143] Not Found: /apple-touch-icon-precomposed.png
[02/Aug/2016 02:49:32] WARNING [django.request:143] Not Found: /favicon.ico
[02/Aug/2016 02:49:32] WARNING [django.request:143] Not Found: /apple-touch-icon.png
[02/Aug/2016 02:49:33] WARNING [django.request:143] Not Found: /apple-touch-icon-precomposed.png
[02/Aug/2016 02:49:33] WARNING [django.request:143] Not Found: /apple-touch-icon.png
[02/Aug/2016 03:18:58] WARNING [django.request:143] Not Found: /favicon.ico
[02/Aug/2016 03:20:28] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 03:20:29] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 03:53:53] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 03:59:13] WARNING [django.request:143] Not Found: /MCFEA/MCFEA/
[02/Aug/2016 04:48:09] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 04:50:20] WARNING [django.request:143] Not Found: /favicon.ico
[02/Aug/2016 04:53:57] ERROR [django.request:231] Internal Server Error: /simulate_tool/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 335, in simulate_tool
    template = loader.get_template('simulate_tool.html')
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 288, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 344, in invalid_block_tag
    raise self.error(token, "Invalid block tag: '%s'" % command)
TemplateSyntaxError: Invalid block tag: 'black'
[02/Aug/2016 05:12:16] ERROR [django.request:231] Internal Server Error: /
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/mgi/views.py", line 48, in home
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 288, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 343, in invalid_block_tag
    (command, get_text_list(["'%s'" % p for p in parse_until])))
TemplateSyntaxError: Invalid block tag: 'static', expected 'endblock'
[02/Aug/2016 05:32:28] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 07:02:48] WARNING [django.request:143] Not Found: /apple-touch-icon-120x120-precomposed.png
[02/Aug/2016 07:02:48] WARNING [django.request:143] Not Found: /apple-touch-icon-120x120.png
[02/Aug/2016 07:02:48] WARNING [django.request:143] Not Found: /apple-touch-icon.png
[02/Aug/2016 07:02:49] WARNING [django.request:143] Not Found: /apple-touch-icon.png
[02/Aug/2016 07:02:49] WARNING [django.request:143] Not Found: /apple-touch-icon-precomposed.png
[02/Aug/2016 07:02:49] WARNING [django.request:143] Not Found: /favicon.ico
[02/Aug/2016 13:53:24] WARNING [django.request:143] Not Found: /favicon.ico
[02/Aug/2016 19:20:51] WARNING [django.request:143] Not Found: /apple-touch-icon-precomposed.png
[02/Aug/2016 19:20:51] WARNING [django.request:143] Not Found: /apple-touch-icon.png
[02/Aug/2016 19:21:57] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 19:30:03] WARNING [django.request:143] Not Found: /favicon.ico
[02/Aug/2016 19:30:03] WARNING [django.request:143] Not Found: /apple-touch-icon-precomposed.png
[02/Aug/2016 19:30:04] WARNING [django.request:143] Not Found: /apple-touch-icon.png
[02/Aug/2016 19:33:57] WARNING [django.request:143] Not Found: /dashboard
[02/Aug/2016 20:11:11] ERROR [django.request:231] Internal Server Error: /curate/select-template
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/admin_mdcs/models.py", line 117, in wrapper
    return view_func(request, *args, **kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/views.py", line 63, in index
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 288, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 343, in invalid_block_tag
    (command, get_text_list(["'%s'" % p for p in parse_until])))
TemplateSyntaxError: Invalid block tag: 'submenu', expected 'endblock'
[02/Aug/2016 20:16:02] ERROR [django.request:231] Internal Server Error: /curate/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/admin_mdcs/models.py", line 117, in wrapper
    return view_func(request, *args, **kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/views.py", line 63, in index
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'admin_menu' appears more than once
[02/Aug/2016 20:16:18] ERROR [django.request:231] Internal Server Error: /curate/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/admin_mdcs/models.py", line 117, in wrapper
    return view_func(request, *args, **kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/views.py", line 63, in index
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'admin_menu' appears more than once
[02/Aug/2016 20:16:33] ERROR [django.request:231] Internal Server Error: /curate/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/admin_mdcs/models.py", line 117, in wrapper
    return view_func(request, *args, **kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/views.py", line 63, in index
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'admin_menu' appears more than once
[02/Aug/2016 20:16:57] ERROR [django.request:231] Internal Server Error: /curate/
Traceback (most recent call last):
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/NANOMINE/Develop/mdcs/admin_mdcs/models.py", line 117, in wrapper
    return view_func(request, *args, **kwargs)
  File "/home/NANOMINE/Develop/mdcs/curate/views.py", line 63, in index
    return HttpResponse(template.render(context))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 148, in render
    return self._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 126, in render
    return compiled_parent._render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 142, in _render
    return self.nodelist.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 844, in render
    bit = self.render_node(node, context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 858, in render_node
    return node.render(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 104, in render
    compiled_parent = self.get_parent(context)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 101, in get_parent
    return get_template(parent)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 144, in get_template
    template, origin = find_template(template_name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 132, in find_template
    source, display_name = loader(name, dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 44, in __call__
    return self.load_template(template_name, template_dirs)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 50, in load_template
    template = get_template_from_string(source, origin, template_name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader.py", line 156, in get_template_from_string
    return Template(source, origin, name)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 132, in __init__
    self.nodelist = compile_string(template_string, origin)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 162, in compile_string
    return parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 201, in do_extends
    nodelist = parser.parse()
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 175, in do_block
    nodelist = parser.parse(('endblock',))
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/base.py", line 290, in parse
    compiled_result = compile_func(self, token)
  File "/home/NANOMINE/anaconda2/lib/python2.7/site-packages/django/template/loader_tags.py", line 171, in do_block
    raise TemplateSyntaxError("'%s' tag with name '%s' appears more than once" % (bits[0], block_name))
TemplateSyntaxError: 'block' tag with name 'admin_menu' appears more than once
